crates/walrus-service/src/node.rs

```diff
@@ -2551,6 +2543,23 @@ impl StorageNodeInner {
         (summary, detail)
     }
 
+    async fn verify_sliver_against_metadata(
+        &self,
+        metadata: VerifiedBlobMetadataWithId,
```

Comment:
Question: Couldn't this take a reference or an `Arc`? Then we wouldn't have to clone it when calling this function.

Reply:
now using an Arc.